src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements Area, Drawable {
```

Comment:
Avoid beginning the class implementation with an empty line. According to the checklist, there shouldn't be an empty line at the start of a class or method body.